modules/compute/vm-instance/main.tf

```diff
@@ -85,6 +85,12 @@ data "google_compute_image" "compute_image" {
   project = var.instance_image.project
 }
 
+resource "null_resource" "image" {
+  triggers = {
+    image = join(",", [var.instance_image.family, var.instance_image.project])
```

Comment:
I think it would be more clear to define two separate triggers and include `validation` safeguards against null or empty string values in variables.tf:

```hcl
resource "null_resource" "image" {
  triggers = {
    family = var.instance_image.family
    project = var.instance_image.project
  }
}
```

Reply:
good point. Thanks!